amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseManagementFactory.java

```diff
@@ -515,6 +516,29 @@ public ShardSyncTaskManager createShardSyncTaskManager(MetricsFactory metricsFac
                 metricsFactory);
     }
 
+    /**
+     * Create ShardSyncTaskManager from the streamConfig passed
+     *
+     * @param metricsFactory - factory to get metrics object
+     * @param streamConfig - streamConfig for which ShardSyncTaskManager needs to be created
```

Comment:
Nit: Missed the param for the new provider

Reply:
done